test/heroku-cli/test.sh

```diff
@@ -5,5 +5,6 @@ set -e
 source dev-container-features-test-lib
 
 check "heroku existance" heroku version
+check "heroku CLI version is equal to 9.3.1" sh -c "heroku version | grep '9.3.1'"
```

Comment:
Move this line to `test_non_latest_version.sh`, so it's checked for `test_non_latest_version` scenario

Reply:
oops lol, that was a dumb mistake